### ISSUE TITLE
Bluetooth audio latency

### ISSUE THREAD

**Issue by Elendiar** [bttv-android/bttv#691]
When watching streams with bluetooth headphones, there is a noticeable audio latency, about 150-250 ms. 
When watching in another app, youtube, or another twitch client, there is no such delay

**Comment by Zerwin** [comment]
I don't think we change anything about bluetooth, so it might just be a base bug that should be fixed via a newer rebase.

**Comment by Zerwin** [comment]
@Elendiar can you check if this still occurs on the 19.0.1 version we rebased on recently ?

**Comment by Elendiar** [comment]
> @Elendiar can you check if this still occurs on the 19.0.1 version we rebased on recently ?

Sadly this latency still occur

**Comment by armantabrizi** [comment]
Hey. The audio delay has been there forever, and it mostly happens in BTTV; the problem is, it used to be fixable by changing Exo2_on_64bit_device and Exo2_on_32bit_device from "reality!!" to "active_12.7", but now, I can't even access the developer tools. it's just gone.

**Comment by bpx-bpx** [comment]
This might be a stupid suggestion coming from someone who knows next to nothing about coding: 
Would it be possible to implement a feature that allows you to delay the video output of the stream so it syncs up again with the delayed audio? If that is generally possible, it could be as "easy" as letting us set a ms value. Or even more user friendly something similar to a "metronome" to sync it up. I heard that Samsung devices natively comes with something called "bluetooth metronome" that allows you to sync up audio and video. So maybe something like that.
